actions/entity/delete.php

```diff
@@ -28,9 +26,12 @@
 $subtype = $entity->getSubtype();
 $container = $entity->getContainerEntity();
 
+$response_data = [
+	'entity' => (array) $entity->toObject(),
```

Comment:
`\Elgg\Http\ResponseFactory` will normalize entities to arrays before serializing them into JSON. In this particular case, I want to avoid problems with calling `toObject()` on a deleted entity, but I still want to send data back to the client.